### PR TITLE
 Update react

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "watch:css": "autoless --autoprefix \"> 5%, last 2 versions, android >= 4.0\" ./src ./build/css",
     "test": "npm run test:lint",
     "test:lint": "eslint --ext .js,.jsx src/",
-    "test:unit": "mocha -R spec ./test/unit/*.js",
```
Comment (gvn) [inline package.json:21]: This must have been copied over from another manifest. There aren't any tests here...

**Comment by gvn** [comment]
Note: I unbundled React and React DOM because they were getting packaged twice by webpack.

Follow on ticket: https://github.com/mozilla/webmaker-core/issues/748

**Comment by gvn** [comment]
Local test URLs:
- http://localhost:8000/#/thumbnail?user=1&project=1&page=1
- http://localhost:8000/#/legal
- http://localhost:8000/
- http://localhost:8000/#/project?user=1&project=1
- http://localhost:8000/#/givemea404

**Comment by gvn** [comment]
Note: On that sample project you'll get a warning "Warning: Failed propType: Invalid prop `targetProjectId` of type `number` supplied to `Link`, expected `string`. Check the render method of `BasicElement`."

This is due to legacy data in the db...

**Comment by alanmoo** [comment]
@gvn Does a db migration have to happen to prevent that or is it only something that will happen in staging?

**Comment by alanmoo** [comment]
Also on http://localhost:8000/#/thumbnail?user=1&project=1&page=1 I'm getting a 404 in the console as it looks for flag.svg. I'm guessing that's also just a project data issue?

File: views/project.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
     <link rel="stylesheet" href="./css/normalize.css">
     <link rel="stylesheet" href="./css/main.css">
     <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Bitter|Roboto|Pacifico">
```
Comment (alanmoo) [inline views/project.html]: This should be `https`, no?

**Comment by gvn** [comment]
The warning should only show up for really old projects. DB migration isn't really worth it since it doesn't cause any functional issues.

The flag.svg has been missing for a while...it's not actually used in desktop (it's for setting link destination). We should probably fix this, but in another issue.

Good call on https...I'll tweak that.

**Comment by xmatthewx** [comment]
victory :v: :boom:

**Pull request merged**